### PR TITLE
Nightwatch check

### PR DESCRIPTION
**Pull request by adambraimbridge**
cc @keirog

### CODE REVIEW

File: src/server/checks/nightwatch.check.js [hunk 1]
```diff
@@ -46,58 +45,84 @@ class NightWatchCheck extends Check {
...
+			// their test names don't equal the session[name]. (e.g. "Barrier Anon User" !== "Barrier")
+			let relevantTests = saucelabsResults.reduce((returnArray, test) => {
+				if (test.name && test.name.toLowerCase().indexOf(nightwatchCheck.session.toLowerCase()) !== -1) {
+					returnArray.push(test);
+				}
+				return returnArray;
+			}, []);
+
+			// Case: No relevant tests were found
+			if (!relevantTests || (typeof relevantTests === 'Array' && relevantTests.length <= 0)) {
```
Comment (keirog) [inline src/server/checks/nightwatch.check.js]: There is no Array primitive in JS, so this won’t work unfortunately.
You could use `Array.isArray(relevantTests)` here, although I normally just go for the slightly less defensive `if (!relevantTests || !relevantTests.length)`

File: config/paywall.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
 			"businessImpact" : "No barriers will be served",
 			"technicalSummary" : "Runs automated browsers tests covering barriers functionality",
 			"panicGuide" : "Don't Panic",
-			"interval" : "5m"
+			"interval" : "2h"
```
Comment (keirog) [inline config/paywall.js]: Is this an agreed interval? Seems a bit infrequent, no?

File: src/server/checks/nightwatch.check.js (hunk 1 above)
Comment (adambraimbridge) [inline src/server/checks/nightwatch.check.js]: Ah yes of course /facepalm

**Comment by keirog** [comment]
LGTM

**Pull request merged**